_studio/shared/asc/include/asc_common_impl.h

```diff
@@ -178,7 +178,7 @@ static inline __m256i LoadPartialYmm(unsigned char *pSrc, mfxI32 len)
 #pragma warning(disable:4505)
```

Comment:
If I read https://msdn.microsoft.com/en-us/library/z85eyax0.aspx correctly, your are fixing exactly this warning. Thus, I believe that you need to check and remove these pragmas as well...